easybuild/easyconfigs/o/OpenForceField-Toolkit/OpenForceField-Toolkit-0.16.0-foss-2023a.eb

```diff
@@ -0,0 +1,144 @@
+easyblock = 'PythonBundle'
+
+name = 'OpenForceField-Toolkit'
+version = '0.16.0'
+
+homepage = 'https://github.com/openforcefield/openff-toolkit'
+description = """The Open Force Field Toolkit provides implementations of the SMIRNOFF format, 
+parameterization engine, and other tools."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+    
+builddependencies = [('poetry', '1.5.1')]
+dependencies = [
+    ('Python', '3.11.3'),
+    ('Python-bundle-PyPI', '2023.06'),
+    ('SciPy-bundle', '2023.07'),
+    ('networkx', '3.1'),
+    ('typing-extensions', '4.9.0'),
+    ('pydantic', '2.5.3'),
+    ('tqdm', '4.66.1'),
+    ('OpenMM', '8.0.0'),
+    ('PyYAML', '6.0'),
+    ('RDKit', '2024.03.3'),
+    ('MDTraj', '1.9.9'),
```

Comment:
Thank you for this contribution - I already used it and installed it! 
It works like a Charm - I used MDTraj 1.10 and made a user happy.

LGTM